### DISCUSSION TITLE
Multi-Lines

### DISCUSSION THREAD

**Discussion by Wexxorz** [runelite/runelite#17975] [Ideas]
With the new DMM coming out, it would be nice to have the Plugin Hub _"Wilderness Lines"_ updated across the whole map.
It would be usefull for Breaches all around Gielinor.
Im not that good at programing but I can help if needed, I have lots of time.

Here's all the Multi spots : https://oldschool.runescape.wiki/images/Multicombat_map.png?95eeb

Here's an example of what it could look like? :
![Multi](https://github.com/runelite/runelite/assets/128922695/96923d31-640c-49fe-830e-271ed3561a64)

**Comment by Nightfirecat** [comment]
I wrote the Wilderness Lines plugin and, while I support the plugin's code being adapted into a broader "Multi Lines" plugin, I'm not interested in trying to create or maintain it myself.

**Comment by DapperMickie** [comment]
@Wexxorz What is your Discord? I'm wanting to at the very least investigate what needs to be done to make this work. I'd love to finish this plugin before Seasonal DMM comes out. 

You can add me: dappermickie on Discord
